uvdat/core/tasks/chart.py

```diff
@@ -45,6 +45,9 @@ def convert_chart(chart_id, conversion_options):
 
 def get_gcc_chart(dataset, context_id):
     chart_name = f'{dataset.name} Greatest Connected Component Sizes'
+    max_nodes = 0
+    for network in dataset.networks.all():
+        max_nodes += network.nodes.count()
```

Comment:
You'll need to also import the `NetworkNode` model, but this will make just one query:

```suggestion
    max_nodes = NetworkNode.objects.filter(network__dataset=dataset).count()
```